internal/cue/flipt.cue

```diff
@@ -0,0 +1,65 @@
+//namespace: string & =~"^[-_,A-Za-z0-9]+$" | *"default"
```

Comment:
```suggestion
namespace?: string & =~"^[-_,A-Za-z0-9]+$" | *"default"
```

Validation will still pass if a user includes and malforms the namespace. This will make `namespace` optional and enforce validation on it if the user chooses to include it.